src/ui/components/Library/Team/View/NewTestRuns/TestRunSpecDetails.tsx

```diff
@@ -54,25 +52,79 @@ export function TestRunSpecDetails() {
   );
 }
 
+interface ErrorCount {
+  message: string;
+  summary: string;
+  count: number;
+}
+
+const getSummary = (message: string) => {
+  const firstLine = message.split("\n")[0];
+
+  if (firstLine.match(/^.*\dms: (.*)/)?.[1]) {
+    return firstLine.match(/^.*\dms: (.*)/)![1];
+  } else {
+    return firstLine;
+  }
+};
+
 function Errors({ failedTests }: { failedTests: TestRunTestWithRecordings[] }) {
+  const { spec } = useContext(TestRunsContext);
+
+  const sortedErrors = useMemo(() => {
+    const errors = failedTests.flatMap(t => t.errors || []);
+    const uniqueErrors = errors.reduce((acc, e) => {
+      const existingError = acc.find(a => a.message === e);
+
+      if (existingError) {
+        existingError.count += 1;
+      } else {
+        acc.push({ message: e, count: 1, summary: getSummary(e) });
```

Comment:
Grouping by the full error but displaying the summary can lead some weird visuals like the failing auth test in https://devtools-git-jaril-scs-1716-show-unique-errors-recordreplay.vercel.app/team/dzowNDAyOGMwYS05ZjM1LTQ2ZjktYTkwYi1jNzJkMTIzNzUxOTI=/runs/ca652ee1-a02f-4e44-9e15-29736c2d4f70 in which the full message is different but the summary is the same.

<img width="454" alt="image" src="https://github.com/replayio/devtools/assets/788456/3fab63ef-b09a-470b-818f-23cd20e017c0">

I'm not sure what the right approach here would be though 🤔

Reply:
Yeah I saw those and wasn't sure either, since usually the difference is the call stack. For now, I think this is a decent compromise until we figure out something better.